RadixWallet/Features/SettingsFeature/SettingsRow.swift

```diff
@@ -0,0 +1,115 @@
+import SwiftUI
+
+// MARK: - SettingsRow
+enum SettingsRow<Feature: FeatureReducer> {
+	/// A standard tappable row with the details specified on the `Model`
+	case model(Model)
+
+	/// A custom row with its own UI. Useful, for example, when we want a `ToggleView`.
+	case custom(AnyView, id: String)
+
+	/// A small row acting as a section header with the provided title.
+	case header(String)
+
+	/// Similar to the `.header`, but with no title.
+	case separator
+
+	@ViewBuilder
+	func build(viewStore: ViewStoreOf<Feature>) -> some View {
```

Comment:
I would prefer this to not be a property on the model, I want the model to just represent the idea of the thing. So I think this enum should be called SettingsRowModel, and then you have SettingsRow which is a View that is created with a model. Or something along those lines. As long as the model is more separate from the view!